docker-build-images/centos7-build.Dockerfile-ppc64le

```diff
@@ -0,0 +1,22 @@
+FROM ppc64le/centos:7
+MAINTAINER Shaun Crampton <shaun@tigera.io>
+ENV STREAM el7
+
+ARG UID
+ARG GID
+
+# Some commands that would typically be run at container build time must be run in a privileged container.
```

Comment:
Unfortunately the entrypoint solution wont solve my problem. I hit the same permissions issue running yum in a non-privileged container (or from a docker build). To make the entrypoint solution work I need to install sudo during the container build.
$ docker run ppc64le/centos:7 yum
error: db5 error(1) from dbenv->open: Operation not permitted
---------
this works ok.
$ docker run --privileged ppc64le/centos:7 yum

I have not resolved the root of this problem, it only occurs in a ubuntu ppc64le VM and only with the centos container.  For now I need to stick with my workaround.